httpx/utils.py

```diff
@@ -64,3 +66,12 @@ def guess_json_utf(data: bytes) -> typing.Optional[str]:
             return "utf-32-le"
         # Did not detect a valid UTF-32 ascii-range character
     return None
+
+
+def get_netrc_login(url: str) -> typing.Union[typing.Tuple[str, str, str], None]:
+    try:
+        _netrc = netrc.netrc(os.environ.get("DEFAULT_NETRC_PATH"))  # type: ignore
```

Comment:
I'd probably prefer a variable name that's not underscore prefixed here. `netrc_info` maybe?

Reply:
Is `DEFAULT_NETRC_PATH` a standardized thing? (Or used by requests or something?)

Reply:
I made it up to explain the initial path.

Reply:
I'm pretty sure `NETRC` is the env variable to specify the path